src/transaction/txParser.ts

```diff
@@ -213,7 +277,11 @@ const parseUnsignedTx = ([txBody, meta]: _UnsignedTxDecoded): _UnsignedTxParsed
     withdrawals,
     metaDataHash,
     meta,
+    validityIntervalStart,
+    // mint, // unsupported in current version
   }
+
+  return removeNullFields(parsedUnsignedTx) as _UnsignedTxParsed
```

Comment:
Is there a reason why we have to removeNullFields at this point since we have to it in `trezorCryptoProvider` again anyway?

Reply:
If you try to pass `validityIntervalStart: undefined` to trezor, it fails. It has to be removed to return signed tx.

Reply:
Refactored.